doc/nrf/libraries/networking/azure_iot_hub.rst

```diff
@@ -372,9 +374,9 @@ Provisioning certificates
 
 The Azure IoT Hub library requires provisioning of the following certificates and a private key for a successful TLS connection:
 
-1. `DigiCert Global Root G2`_ - The root CA certificate, used to verify the server's certificate chain while connecting.
-#. `Baltimore CyberTrust Root Certificate`_ - Azures legacy root CA certificate, needed to verify the Azure server's  that haven't migrated to `DigiCert Global Root G2`_ yet.
-#. Public device certificate - generated by the procedures described in REMOVED , used by Azure IoT Hub to authenticate the device.
+1. `DigiCert Global Root G2`_ - The root CA certificate used to verify the server's certificate chain while connecting.
+#. `Baltimore CyberTrust Root Certificate`_ - Azure`s legacy root CA certificate needed to verify the Azure server's  that have not migrated to `DigiCert Global Root G2`_ yet.
```

Comment:
I think we should use apostrophe here instead of backtick. Probably neutral `'` or right `’`.
```suggestion
#. `Baltimore CyberTrust Root Certificate`_ - Azure’s legacy root CA certificate needed to verify the Azure server's  that have not migrated to `DigiCert Global Root G2`_ yet.
```